src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,47 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
-    public void getStatistic(String fromFileName, String toFileName) {
+    public static final String supply = "supply";
+    public static final String buy = "buy";
+    public static final int ZERO_LENGTH = 0;
+    public static final int firstElement = 0;
+    public static final int secondElement = 1;
 
+    public void getStatistic(String fromFileName, String toFileName) {
```

Comment:
As described in common mistakes, you should divide your logic into separate methods. I'd recommend creating separate methods for reading, report generating, and writing